config/mcus/octopus.cfg

```diff
@@ -1,45 +0,0 @@
-[mcu]
-##--------------------------------------------------------------------
-serial: /dev/serial/by-id/change-me-to-the-correct-mcu-path

```

Comment:
You deleted the original file in your commit. Can you bring it back please ?

Reply:
Oh sorry this was by mistake. But because of shared PIN definitions between the CAN Board and the Octopus it's needed to have a second definition for the CAN config